docs/app/Examples/elements/Flag/Types/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+import { Message } from 'stardust'
+
+export default class FlagTypesExamples extends Component {
+  render() {
+    return (
+      <ExampleSection title='Types'>
+        <ComponentExample
+          title='Flag'
+          description='A flag can use the two digit country code, the full name, or a common alias.
+          Make sure you use lowercase letters.'
```

Comment:
Let's remove this sentence as propType validation will handle that.  Lower case letters are also shown in the example code itself.

```diff
-          Make sure you use lowercase letters.
```